### ISSUE TITLE
v0.21

### ISSUE THREAD

**Issue by AdriVanHoudt** [zaggino/brackets-eslint#13]
Can we have an update to 0.21? I want to use the new extends feature.

**Comment by zaggino** [comment]
Done, btw with `brackets-npm-registry` doing `reinstall` should be enough because dependencies in `brackets-eslint` are configured to install the latest version available.

**Comment by AdriVanHoudt** [comment]
thanks! but you hardcode the version right? https://github.com/zaggino/brackets-eslint/blob/master/package.json#L29

**Comment by zaggino** [comment]
Ah you're right, it shouldn't be `^0.21.0` but `^0.21` or something like that. Maybe I'll just put `latest` in there.

**Comment by AdriVanHoudt** [comment]
http://semver.npmjs.com/ ;) I tend to use .x notation like 1.x.x, but in this case you would go with ^0, this exludes alpha and dev versions and doesn't auto switch to the soon to come v1.0.0

**Comment by AdriVanHoudt** [comment]
or 0.x.x whatever you suits you

**Comment by zaggino** [comment]
Yes, I'll fix that ;)

**Comment by AdriVanHoudt** [comment]
hmm I'm having an issue where the plugin from the base config which a extend doesn't work. I tested it with command line and that seems to pick it up. Probably has to do with how you detect plugins.

**Comment by zaggino** [comment]
I'll need more info on that, which plugin, config and a sample of code where it should show an error :)

**Comment by AdriVanHoudt** [comment]
Base config is from Lab https://github.com/hapijs/lab/blob/master/lib/linters/eslint/.eslintrc with `eslint-plugin-hapi` https://github.com/cjihrig/eslint-plugin-hapi

Code sample: 
```js
var f = function() {
    console.log('a');
};
```
Should give warning that an empty line is needed after `function` like
```js
var f = function() {

    console.log('a');
};
```

**Comment by AdriVanHoudt** [comment]
I do have the plugin globally but that probably doesn't work for plugins + extends feature

**Comment by zaggino** [comment]
I've published `0.21.1` and now it's working fine for me:
![image](https://cloud.githubusercontent.com/assets/1067319/7673947/45e81e10-fd60-11e4-8cb4-611bb8d3546f.png)

only thing I had to do to test in my project was:
```
npm install -g eslint-plugin-hapi
npm install lab
```
and added this to `.eslintrc` config:
```
extends: "./node_modules/lab/lib/linters/eslint/.eslintrc"
```

**Comment by AdriVanHoudt** [comment]
nice, thanks, everything works now! thanks for the efford
